colander/tests/test_colander.py

```diff
@@ -1557,6 +1557,13 @@ def test_serialize_string_with_high_unresolveable_high_order_chars(self):
         e = invalid_exc(typ.serialize, node, not_utf8)
         self.assertTrue('cannot be serialized' in e.msg)
 
+    def test_serialize_encoding_with_non_string_type(self):
+        utf8 = '123'.encode('utf-8')
```

Comment:
Sorry, I develop everything in Python 3 and then deal with Python 2 only when I need backwards compatibility.  I know this is already doing what your describing in Py3, but I don't understand how it's not doing this in Py2.  In Py2, should this be `u'123'.encode('utf-8')` to be explicit?